src/zenml/integrations/tensorflow/__init__.py

```diff
@@ -31,14 +34,20 @@ def activate(cls) -> None:
         """Activates the integration."""
         # need to import this explicitly to load the Tensorflow file IO support
         # for S3 and other file systems
-        if (
-            not platform.system() == "Darwin"
-            or not platform.machine() == "arm64"
-        ):
+        if not platform.system() == "Darwin" or not platform.machine() == "arm64":
             import tensorflow_io  # type: ignore
 
         from zenml.integrations.tensorflow import materializers  # noqa
 
+        if sys.version_info.minor == 8:
+            logger.warning(
+                "Python 3.8 with TensorFlow is not fully "
+                "compatible with Pydantic 2 requirements. "
+                "Consider upgrading to a higher Python "
+                "version, if you would like to use "
```

Comment:
```suggestion
                "version if you would like to use the "
```

Reply:
I was wondering if we have this error message when someone does `zenml integration install tensorflow` (and/or).

Reply:
Not really, if I plug it into requirements it is too annoying since requirements are being read in many places. This one will pop-up on the run of a pipeline. Do you think I need it in the CLI command as well?